packages/nimble-components/src/table/components/row/styles.ts

```diff
@@ -10,10 +10,13 @@ export const styles = css`
     ${display('grid')}
 
     :host {
-        height: ${controlHeight};
+        height: calc(${controlHeight} + 2 * ${borderWidth});
         border-top: calc(2 * ${borderWidth}) solid ${tableRowBorderColor};
         grid-auto-flow: column;
         grid-auto-columns: 1fr;
+        grid-template-columns: inherit;
```

Comment:
To follow up on this comment:

> But actually we shouldn't need it anymore, now that the grid columns are configured as a CSS custom property and CSS custom properties flow through shadow roots, you can use the property set in the higher-level table directly in the stylesheet of this child element, you don't need to do this inherit forwarding stuff.

Can we avoid the inherit and use the css custom property instead? Inheriting layout is usually not expected.

ie something like:

```suggestion
        grid-template-columns: var(--ni-private-table-row-grid-columns) auto;
```